guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -10,26 +10,53 @@ redirect_from:
-{% assign unsecure_base_url = "{{unsecure_base_url}}" %}
--->
-
-The `{{site.data.var.ct}}` package is compatible with {{site.data.var.ee}} version 2.1.4 and later to provide a rich set of features you can use to manage your {{site.data.var.ece}} project. It contains scripts and {{site.data.var.ece}} commands designed to help manage your code and automatically build and deploy your projects.
-
-You can list the available `{{site.data.var.ct}}` commands using:
-
-```bash
-php ./vendor/bin/ece-tools list
-```
-
 The following updates describe the latest improvements to the `{{site.data.var.ct}}` package, which uses the following version sequence: `200<major>.<minor>.<patch>`. See [Upgrades and patches]({{ site.baseurl }}/guides/v2.1/cloud/project/project-upgrade-parent.html) for information about updating to the latest release of the `{{site.data.var.ct}}` package.
 
 The release notes include:
 
 -   {:.new}New features
 -   {:.fix}Fixes and improvements
 
+## v2002.0.20
+
+-   {:.new}**Docker Updates**—
+
+    -   {:.new}<!-- MAGECLOUD-3129/3684 -->You can now perform functional testing using the `{{site.data.var.ct}}` package in the Docker environment. See [Functional testing in Docker]({{page.baseurl}}/cloud/docker/docker-development-testing.html).
+
+    -   {:.new}<!-- MAGECLOUD-3357 -->Added support for configuring PHP modules using the `.magento.app.yaml` file. Any [PHP Extensions specified in the `.magento.app.yaml` file]({{page.baseurl}}/cloud/project/project-conf-files_magento-app.html#php-extensions) become available in the Docker PHP containers.
+
+    -   {:.fix}<!-- MAGECLOUD-3582 -->Corrected the default path for Magento root when using the Docker environment. Previously, the Magento root was accessible in the `var/www/magento` directory. Now, the Magento root is in the `/app` directory, as expected.
+
+    -   {:.fix}<!-- MAGECLOUD-3832 -->Updated the Sodium library from version 1.0.11 to version 1.0.18, and updated the Sodium PHP extension.
+
+    -   {:.fix}<!-- MAGECLOUD-3446 -->Added the `analysis-icu` and the `analysis-phonetic` Elasticsearch plugins to all Docker images.
+
+-   {:.new}**Environment variable updates**—
+
+    -   {:.new}<!-- MAGECLOUD-2901 -->Added support for database table prefixes using the [DATABASE_CONFIGURATION environment variable]({{page.baseurl}}/cloud/env/variables-deploy.html#database_configuration).
+
+    -   {:.fix}<!-- MAGECLOUD-3258 -->You can configure the **WARM_UP_PAGES** environment variable to cache single pages, multiple domains, and multiple pages. See the expanded definition in the [post-deploy variables]({{page.baseurl}}/cloud/env/variables-post-deploy.html#warm_up_pages) content.
+
+    -   {:.new}<!-- MAGECLOUD-3602 -->Added the **FORCE_UPDATE_URLS** deploy variable to update Magento base URLs when deploying to Pro and Starter production and staging environments. See the definition in the [deploy variables]({{page.baseurl}}/cloud/env/variables-deploy.html#force_update_urls) content.
```

Comment:
Shouldn't all "new" line items be grouped together?

Reply:
Not really. With the bullets, I don't see us worrying about it. Our release list is not typically long and we usually group like things, such Docker changes, Environment variable changes. And since we have three versions of this, the notes are not always the same.

Reply:
I resorted the notes. Does that help for emphasis? I also added a Bolding for the new command that was submitted by a contributor.